pixart.js

```diff
@@ -1 +1,13 @@
-
+$(document).ready(function(){
+  $("#set-color").on("click", function(e){
+    e.preventDefault();
+    console.log("test!");
+    $(".brush").css("background-color", $("#color-field").val());
```

Comment:
Might be good to extract the `$("#color-field").val()` into a variable just to make this line more readable, e.g.:

```js
var newColor = $("#color-field").val();
$(".brush").css("background-color", newColor);
```